Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -235,9 +239,8 @@ public void deleteIndexes() throws LuceneException {
 
             for (IWordID wordId : wordIdds) {
                 IWord word = dict.getWord(wordId);
-                List<ConceptEntry> entries = databaseClient.getConceptByWordnetId(word.getID().toString());
-                if (entries != null && !entries.isEmpty()) {
-                    logger.debug("Found concept for " + wordId.toString());
+                if (wordnetIdsOfWrappers.contains(word.getID().toString())) {
```

Comment:
Hm, the method "getConceptByWordnetId" should always return a concept if there is a concept that wraps the provided wordnet id. ```getConceptByWordnetId``` should be changed to do that then.